packages/cli/package.json

```diff
@@ -128,13 +130,11 @@
     "semver": "^7.5.3",
     "style-loader": "^3.3.1",
     "sucrase": "^3.20.2",
-    "swc-loader": "^0.2.3",
     "tar": "^6.1.12",
     "terser-webpack-plugin": "^5.1.3",
     "tsx": "^4.0.0",
     "util": "^0.12.3",
-    "webpack": "^5.70.0",
-    "webpack-dev-server": "^4.7.3",
+    "webpack": "^5.90.1",
```

Comment:
It's still peer dependency of some plugin like `fork-ts-checker-webpack-plugin`, I'll raise PRs there for not requiring `webpack` installed.